initial_pose_defaults:
  map1:
    x: -6.804637535308014
    y: -5.079333617726722
    yaw: 0.0
  map2:
    x: 10.0
    y: 1.0
    yaw: 0.0
